src/main/java/duke/CommandParser.java

```diff
@@ -0,0 +1,163 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.command.*;
```

Comment:
This wildcard import might be a coding violation. Consider specifying the imported classes individually (even though I know it might be the case that all classes in that package are used)